ilc/package.json

```
@@ -6,7 +6,9 @@
   "scripts": {
     "test": "cross-env NODE_ENV=test mocha",
     "test:coverage": "cross-env NODE_ENV=test nyc mocha",
+    "test:e2e": "npx codeceptjs run --steps --verbose",
```

Should work w/o npx:
```suggestion
    "test:e2e": "codeceptjs run --steps --verbose",
```